modules/components/AsyncConnect.js

```diff
@@ -44,7 +44,9 @@ export default class AsyncConnect extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    this.loadAsyncData(nextProps);
+    if (!this.context.store.getState().reduxAsyncConnect.loading) {
```

Comment:
Not on initial render, but when initially receiving async props.
It will mount as an empty element first
```
componentDidMount (AsyncConnect.js:55)
invokeComponentDidMountWithTimer (ReactCompositeComponent.js:54)
notifyAll (CallbackQueue.js:67)
close (ReactReconcileT…nsaction.js:81)
closeAll (Transaction.js:204)
perform (Transaction.js:151)
batchedMountComponentIntoNode (ReactMount.js:126)
perform (Transaction.js:138)
batchedUpdates (ReactDefaultBat…Strategy.js:63)
batchedUpdates (ReactUpdates.js:98)
_renderNewRootComponent (ReactMount.js:285)
_renderSubtreeIntoContainer (ReactMount.js:371)
render (ReactMount.js:392)
(anonymous function) (client.js:47)
(anonymous function) (main-458f43fc6a…d041cf.js:1984)
__webpack_require__ (bootstrap 458f4…2eaad041cf:555)
fn (bootstrap 458f4…a2eaad041cf:86)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
__webpack_require__ (bootstrap 458f4…2eaad041cf:555)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
```

And then receive async props
```
componentWillReceiveProps (AsyncConnect.js:61)
updateComponent (ReactCompositeComponent.js:601)
receiveComponent (ReactCompositeComponent.js:534)
receiveComponent (ReactReconciler.js:131)
_updateRenderedComponent (ReactCompositeComponent.js:737)
_performComponentUpdate (ReactCompositeComponent.js:715)
updateComponent (ReactCompositeComponent.js:634)
receiveComponent (ReactCompositeComponent.js:534)
receiveComponent (ReactReconciler.js:131)
_updateRenderedComponent (ReactCompositeComponent.js:737)
_performComponentUpdate (ReactCompositeComponent.js:715)
updateComponent (ReactCompositeComponent.js:634)
performUpdateIfNecessary (ReactCompositeComponent.js:548)
performUpdateIfNecessary (ReactReconciler.js:165)
runBatchedUpdates (ReactUpdates.js:151)
perform (Transaction.js:138)
perform (Transaction.js:138)
perform (ReactUpdates.js:90)
flushBatchedUpdates (ReactUpdates.js:173)
closeAll (Transaction.js:204)
perform (Transaction.js:151)
batchedUpdates (ReactDefaultBat…Strategy.js:63)
batchedUpdates (ReactUpdates.js:98)
_renderNewRootComponent (ReactMount.js:285)
_renderSubtreeIntoContainer (ReactMount.js:371)
render (ReactMount.js:392)
(anonymous function) (client.js:47)
(anonymous function) (main-458f43fc6a…d041cf.js:1984)
__webpack_require__ (bootstrap 458f4…2eaad041cf:555)
fn (bootstrap 458f4…a2eaad041cf:86)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
__webpack_require__ (bootstrap 458f4…2eaad041cf:555)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
(anonymous function) (bootstrap 458f4…2eaad041cf:578)
```